### PR TITLE
Tackle2 backend changes

### PR DESCRIPTION
**Pull request by gildub**
Tackle2 now uses the following:
* `konveyor-tackle` namespace.
* Env variable APPLICATION_INVENTORY_API_URL = `http://tackle-hub:8080` instead of `http://tackle-hub-api:8080`.

### CODE REVIEW

**Review by ibolton336 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,126 @@
...
+```
+git clone https://github.com/konveyor/tackle2-ui
+cd tackle-ui
+npm install
+```
+
+With a Tackle2 environment available (with kubectl authentication validated)  
+then one can start a tackle-ui instance locallly serving the pages from the current source code: 
+
+`npm run start:dev:local`
```
Comment (ibolton336) [inline README.md:20]: need to specify which dir to run this in.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,126 @@
...
+The Express server (pkg/server/setupProxy.js) uses by default the environment variables APPLICATION_INVENTORY_API_URL, CONTROLS_API_URL, PATHFINDER_API_URL and SSO_SERVER_URL to determine the backend endpoints. 
+If no env. variables are defined, the server then listens on ports 9001 (SSO), 9002 (Application inventory and controls) and 9003 (Pathfinder).
+
+In which case the port forwarding must activated to route Tackle Keycloack (SSO), Tackle Hub and Tackle Pathfinder requests.
+To set-up kubectl port forwarding Tackle2 services to localhost, open a terminal and run each following command separatly: 
+`$ kubectl port-forward svc/tackle-keycloak-sso -n konveyor-tackle 9001:8080`
+`$ kubectl port-forward svc/tackle-hub -n konveyor-tackle 9002:8080`
+`$ kubectl port-forward svc/tackle-pathfinder-api -n konveyor-tackle 9003:8080` 
+
+That's exactly what `start:dev:local` takes does as it takes care of port forwarding all tackles2 services for us.
```
Comment (ibolton336) [inline README.md:42]: typo takes/does

File: README.md [hunk 3]
```diff
@@ -1,2 +1,126 @@
...
+Minikube provides a certified Kubernetes platform with most of the vanilla features allowing to simulate a real Kubernetes cluster with a fairly high level of feature parity.
+The installation consists in downloading the minikube binary into ${HOME}/.local/bin, so that we don't need to modify the PATH variable. Other options are described in Minikube documentation.
+
+```
+$ curl -sL -o ${HOME}/.local/bin/minikube \
+    https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+$ chmod u+x ~/.local/bin/minikube
+```
+
+Add ${HOME}/.local/bin to your PATH environment variable
```
Comment (ibolton336) [inline README.md:58]: :+1:

File: README.md [hunk 4]
```diff
@@ -1,2 +1,126 @@
...
+```
+cat << EOF | kubectl apply -f -
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tackle
+  namespace: konveyor-tackle
+spec:
+EOF
+```
```
Comment (ibolton336) [inline README.md:126]: Might be good to include this command for waiting for pods to spin up before running proxy. 

```
kubectl wait deployment tackle-keycloak-sso \                                                                                                                                                                                                   
    -n konveyor-tackle --for condition=Available --timeout=5m

```

**Review by ibolton336 — APPROVED** [review]

**Comment by ibolton336** [comment]
Gilles- Changes tested and look fine. IF you find time, update the small changes I listed above. Thx!

File: README.md (hunk 1 above)
Comment (gildub) [reply README.md:20]: Right we need that until we have npm workspace (hopefully soon) in place.

File: README.md (hunk 2 above)
Comment (gildub) [reply README.md:42]: Noted

File: README.md (hunk 4 above)
Comment (gildub) [reply README.md:126]: Good point, this one is really handy.

**Pull request merged**